rs/nns/integration_tests/src/neuron_following.rs

```diff
@@ -537,6 +539,148 @@ async fn test_prune_some_following() {
+
+        // This is so that original_neurons will match what gets
+        // returned by the governance canister.
+        visibility: Some(Visibility::Private as i32),
+
+        ..Default::default()
+    };
+
+    // When this occurs in the voting_power_refreshed_timestamp_seconds field,
+    // that field is supposed to get set to (the new)
+    // DEFAULT_VOTING_POWER_REFRESHED_TIMESTAMP_SECONDS.
+    const EVIL_TIMESTAMP_SECONDS: u64 = 1731628801;
+    // Assert that evil timestamp is "much greater" than the proper value.
+    {
+        let seconds = EVIL_TIMESTAMP_SECONDS - DEFAULT_VOTING_POWER_REFRESHED_TIMESTAMP_SECONDS;
+        assert!(
+            seconds as f64 / ONE_DAY_SECONDS as f64 > 60.0,
+            "{} vs. {}",
+            EVIL_TIMESTAMP_SECONDS as f64 / ONE_DAY_SECONDS as f64,
+            DEFAULT_VOTING_POWER_REFRESHED_TIMESTAMP_SECONDS as f64 / ONE_DAY_SECONDS as f64,
+        );
+    }
+
+    let voting_power_refreshed_timestamp_seconds_neuron_values = vec![
+        EVIL_TIMESTAMP_SECONDS - 1,
+        EVIL_TIMESTAMP_SECONDS,
+        EVIL_TIMESTAMP_SECONDS + 1,
+        now_seconds,
+    ];
+
+    let cached_neuron_stake_e8s_neuron_values = vec![
+        0,       // inactive -> stable memory
+        42 * E8, // active -> heap
+    ];
+
+    let original_neurons = voting_power_refreshed_timestamp_seconds_neuron_values
+        .into_iter()
+        .cartesian_product(cached_neuron_stake_e8s_neuron_values.into_iter())
+        .collect::<Vec<_>>();
+
+    let controller = PrincipalId::new_user_test_id(482_783_461);
+
+    let ids = 100..;
+    let original_neurons = ids
+        .zip(original_neurons.into_iter())
+        .map(
+            |(id, (voting_power_refreshed_timestamp_seconds, cached_neuron_stake_e8s))| Neuron {
+                id: Some(NeuronId { id }),
+                controller: Some(controller),
+                account: vec![id as u8; 32],
+                cached_neuron_stake_e8s,
+                voting_power_refreshed_timestamp_seconds: Some(
+                    voting_power_refreshed_timestamp_seconds,
+                ),
+
+                ..neuron_base.clone()
+            },
+        )
+        .collect::<Vec<_>>();
+
+    assert_eq!(original_neurons.len(), 8);
+
+    let governance_proto = GovernanceCanisterInitPayloadBuilder::new()
+        .with_additional_neurons(original_neurons.clone())
+        .build();
+
+    install_canister(
+        &pocket_ic,
+        "NNS Governance",
+        GOVERNANCE_CANISTER_ID,
+        governance_proto.encode_to_vec(),
+        // TODO: Once following pruning is released, replace with vanilla
+        // build_governance_wasm(). For now, the feature is only enabled when
+        // built with feature = "test".
+        build_test_governance_wasm(),
+        Some(ROOT_CANISTER_ID.get()),
+    )
+    .await;
+
+    // Step 2: Call the code under test.
+
+    // Wait for backfilling to complete.
+    for _ in 0..100 {
```

Comment:
Nit. 

Perhaps you could reduce the number of ticks to make sure the change propagates not slower than you would expect.